### PR TITLE
refactor fps calculation based on result variation

### PR DESCRIPTION
**Pull request by MikeLien**

### CODE REVIEW

**Comment by MikeLien** [comment]
@ShakoHo r?
@askeing r?
@ypwalter r?

File: lib/helper/resultHelper.py [hunk 1]
```diff
@@ -161,35 +161,26 @@ def fps_cal(file_path, default_fps):
     fps_stat = "0"
```
Comment (askeing) [inline lib/helper/resultHelper.py]: I will suggest using
```
STATUS_SUCCESS = '0'
STATUS_FAIL = '1'
...
fps_stat = STATUS_SUCCESS
```

File: lib/helper/resultHelper.py [hunk 2]
```diff
@@ -161,35 +161,26 @@ def fps_cal(file_path, default_fps):
...
-                float(default_fps_element[0][1] + mode_fps[1]) / total_record >= 0.9:
-            fps_return = round(default_fps_element[0][0] - float(default_fps_element[0][0] - mode_fps[0]) * default_fps_element[0][1] / total_record)
-        else:
-            fps_return = mode_fps[0]
+        all_fps_tuple = count.most_common()
+        if not all_fps_tuple:
+            logger.error("Cannot get fps information from log.")
             fps_stat = "1"
+        else:
+            total_record = sum([item[1] for item in all_fps_tuple])
```
Comment (askeing) [inline lib/helper/resultHelper.py]: adding comment here to record `item[0] is fps` and `item[1] is counter` will be helpful.

**Comment by askeing** [comment]
lgtm r+

**Pull request merged**